### ISSUE TITLE
🔧 Handbrake Profile selection in the settings should be a dropdown menu

### ISSUE THREAD

**Issue by SylvainMT** [automatic-ripping-machine/automatic-ripping-machine#1110]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

Not a problem per se, but the current use of using a text field is error-prone.  Making it difficult for users to know what the profile options are.

### Describe the solution you'd like?

In the Ripper Settings page, the settings;

- HB_PRESET_DVD
- HB_PRESET_BD

Should be a dropdown list.  The dropdown list should be dynamically populated from the results of `HandBrakeCLI -z`.

Further, I would also suggest that a new field called "HB_PRESET_FILE" should be present, it should be a browse field to select a preset file and upload it to the ARM machine for its use.  (The preset file is something that can be easily created using the HandBrake UI, allowing the user some fine-grained control over how Handbrake processes the transcode.)  If the filed is populated, when ARM calls HandBrakeCLI it should add the option: `HandBrakeCLI --preset-import-file={ValueOf_HB_PRESET_FILE}`  The preset list in the settings page could also be prepopulated from the same preset file with `HandBrakeCLI -z --preset-import-file={ValueOf_HB_PRESET_FILE}`

### Describe alternatives you've considered?

I have been doing this manually.  Copy-pasting the name of the HB preset from the terminal directly in the arm.yaml.

I've also been using the `HB_ARGS_DVD` AND `HB_ARGS_BD` field to set a custom preset file.

### Anything else?

HandBrakeCLI Commandline Reference: https://handbrake.fr/docs/en/1.7.0/cli/command-line-reference.html
HandBrakeCLI Commandline Options: https://handbrake.fr/docs/en/1.7.0/cli/cli-options.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by microtechno9000** [comment]
Agree, a lot of the config settings should be a list or fixed so users don't break things. Also makes it easier to configure without reading the wiki

Its on the list I plan to resolve, with 3.0

**Comment by SylvainMT** [comment]
Willing to work on it, once you guys have firmed up the code for V3.0   It was suggested that the issue should be posted so it can be tracked, so that is what I did :)

**Comment by microtechno9000** [comment]
All good, saw the discussion on discord, I should have mentioned.

Adding context here for other users who might see this post.

**Comment by shitwolfymakes** [comment]
Seconded, per the chat on Discord this is a perfect issue to start on as part of 3.1. Best to keep feature additions separate from the total rewrite we're shipping in 3.1

**Comment by iampegram** [comment]
For those looking for a workaround in the meantime, here's what I did to use a custom preset:

1. Save a preset from Handbrake somewhere it can be accessed from inside the container (I chose `/home/arm`) and give it a simple name (I chose `preset.json`)
2. In `HB_PRESET_DVD` and/or `HB_PRESET_BD`, type the exact name of your preset (in my case, `Library Settings 720p30`)
3. In the `HB_ARGS_DVD` and/or `HB_ARGS_BD` field, enter the following arguments using the full path to your preset file (in my case, `--preset-import-file /home/arm/preset.json`)
